nextstrain/cli/remote/s3.py

```diff
@@ -65,6 +65,9 @@
 mimetypes.add_type("application/json", ".json")
 mimetypes.add_type("text/markdown", ".md")
 
+# Add zstd to encodings map since it is not yet included in the standard library
+mimetypes.encodings_map[".zst"] = "zstd"
```

Comment:
_non-blocking, just a comment_

This addition is just fine, but seeing it and the existing additions above it reminds me that this module should probably be instantiating its own [`MimeTypes` instance](https://docs.python.org/3/library/mimetypes.html#mimetypes-objects) and modifying that rather than modifying the shared process-global instance.

Reply:
Tracking in https://github.com/nextstrain/cli/issues/331.